homeassistant/components/dwd_weather_warnings/sensor.py

```diff
@@ -49,7 +47,7 @@
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
-        vol.Optional(CONF_REGION_NAME): cv.string,
+        vol.Required(CONF_REGION_NAME): cv.string,
```

Comment:
Is this a breaking change? It needs to be noted if so.

Reply:
Without an valid value set for CONF_REGION_NAME the sensor (even the current implemantion) isn't working. It doesn't generate an error but it is also not delivering data.

If you look back to the intial pull request a default was set for CONF_REGION_NAME but removed during review but the config wasn't changed from Optional to Required.
https://github.com/home-assistant/core/pull/8657

Reply:
Hmm, okay – so I suppose it isn't breaking anything that wasn't already broken. 😆 Disregard.